sqlmesh/core/model/kind.py

```diff
@@ -755,6 +765,15 @@ def to_expression(
         )
 
 
+class ManagedKind(_ModelKind):
+    name: Literal[ModelKindName.MANAGED] = ModelKindName.MANAGED
+    disable_restatement: SQLGlotBool = True
```

Comment:
Let's make it `t.Literal[True] = True` so that the value can be overridden by a user.

Reply:
Or make it a property method which returns True